1-introduction/1-06_a-further-analogy.md

```diff
@@ -8,7 +8,7 @@ _above: Citymapper website showing possible routes, considering multiple modes o
 
 To help illustrate this analogy, let's make a table comparing the expected activities when searching for the quickest commute route and what their equivalent would be in a generative design process.
```

Comment:
I think this analogy requires a bit more explanation if we're going to use it - in the mapping case we're changing inputs to existing algorithms that are running on the mapping system......in the Refinery case, we're choosing which inputs are going to participate in the study....isn't it pretty different?